### PR TITLE
add deprecation notice in advance of archiving repo

### PR DESCRIPTION
**Pull request by jmcook1186**
The relevant material from ethdocs.org has been migrated over to ethereum.org, enabling ethdocs.org to be sunsetted without losing the valuable information it contains. This PR simply adds a short deprecation notice to the README alerting readers to the imminent deprecation of ethdocs.org.

Relates to issue 5731 in ethereum.org repo [here](https://github.com/ethereum/ethereum-org-website/issues/5731)

### CODE REVIEW

**Review by minimalsm — APPROVED** [review]
LGTM

File: README.rst [hunk 1]
```diff
@@ -9,9 +9,15 @@ Homestead-Guide
...
+==================
+
+The Ethereum Homestead Guide is the reference documentation accompanying the **Homestead release** of the Ethereum project. 
+The information in this repository is largely out of date. Relevant material has been migrated to https://ethereum.org.
+and this repository is **due to be archived shortly**. For current information on Ethereum, please visit https://ethereum.org instead.
+This repository is for historical reference only!
+
 
 HOW YOU CAN HELP
 ================================================================================
```
Comment (samajammin) [inline README.rst:23]:
```suggestion
================================================================================

Looking to contribute to Ethereum documentation? Check out the ethereum.org repo: https://github.com/ethereum/ethereum-org-website

```

File: README.rst [hunk 2]
```diff
@@ -9,9 +9,15 @@ Homestead-Guide
...
+DEPRECATION NOTICE
+==================
+
+The Ethereum Homestead Guide is the reference documentation accompanying the **Homestead release** of the Ethereum project. 
+The information in this repository is largely out of date. Relevant material has been migrated to https://ethereum.org.
+and this repository is **due to be archived shortly**. For current information on Ethereum, please visit https://ethereum.org instead.
+This repository is for historical reference only!
+
 
 HOW YOU CAN HELP
```
Comment (samajammin) [inline README.rst:22]: Perhaps we should remove the other sections on "HOW YOU CAN HELP"? e.g. I don't think we should push people to "Write Content to the Guide" or "Review What Has Been Written".

I suggested a note to push folks to the e.org repo.

**Review by samajammin — CHANGES_REQUESTED** [review]
LGTM! Suggested a couple small updates.

**Pull request merged**